components/sections/call_to_action/variant_d.tsx

```diff
@@ -41,264 +50,197 @@ function VariantD({
                 />
               </Link>
             )}
-            <h1 className="font-heading mb-4 text-4xl font-bold md:text-5xl">
-              {title}
-            </h1>
-            <p className="mb-8 leading-loose text-gray-700">{text}</p>
+            <Heading className="mb-4">{title}</Heading>
```

Comment:
Just like the logo image and other elements, kindly update the Title and Text elements to be returned only when it is defined/there is a value.